plugins/woocommerce-admin/client/marketplace/components/product-list-content/no-results.tsx

```diff
@@ -25,6 +25,8 @@ export default function NoResults(): JSX.Element {
 	useEffect( () => {
 		if ( query.term ) {
 			setNoResultsTerm( query.term );
+
+			return;
```

Comment:
Cem explained the reason for this in a chat. If we don't return here and there's a category, the search term isn't being displayed in the "No results" message at the moment. As the term is more important, he opted to return and show it in this situation. We'll come back and improve this later.